src/modules/CRUD/components/Details/PropertyItem/Type/GeometryType/IconFromCoordinates/IconFromCoordinates.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import geojsonExtent from '@mapbox/geojson-extent';
+
+
+const getCoordinatesFromType = (type, coordinates) => {
+  if (type === 'MultiPolygon') {
+    return coordinates.flat();
+  }
+  if (type === 'LineString') {
+    return [coordinates];
+  }
+  return coordinates;
+};
+
+const IconFromCoordinates = ({ coordinates, type, width, height }) => {
```

Comment:
Can you add comment to explain what this component for ? Should be in TF, i love it.

Reply:
Okay for the comments.
For now, I think to keep it in this project because not mature enough; there are cases like multiGeometry which would not render correctly